src/bin/cht-conf.js

```diff
@@ -3,12 +3,12 @@
 const { error } = require('../lib/log');
```

Comment:
Maybe this file should just be called `index.js` or something?

Reply:
Done!